_episodes/01.md

```diff
@@ -31,7 +30,7 @@ keypoints:
 
 > ## Why GitHub?
 >
-> We will do this exercise on [GitHub](https://github.com) but also [GitLab](https://gitlab.com) and [Bitbucket](https://bitbucket.org) allow similar workflows and basically everything that we will discuss is transferable. With this material and these exercises we do not endorse the company GitHub. We have chosen to demonstrate a number of concepts using examples with GitHub because it is currently the most popular web platform for hosting Git repositories and the chance is high that you will interact with GitHub-based repositories even if you choose to host your Git repository on another platform.
+> We will do this exercise on [GitHub](https://github.com) but also [GitLab](https://gitlab.com) and [Bitbucket](https://bitbucket.org) allow similar ways of working and basically everything that we will discuss is transferable. With this material and these exercises we do not endorse the company GitHub. We have chosen to demonstrate a number of concepts using examples with GitHub because it is currently the most popular web platform for hosting Git repositories and the chance is high that you will interact with GitHub-based repositories even if you choose to host your Git repository on another platform.
 >
 > We also encourage course participants to use CodeRefinery’s [Nordic research software repository platform](https://source.coderefinery.org),
```

Comment:
What is this recommentation, do we still want to keep it?

Reply:
I think we do, I've added an explanation in the text. Apparently it is a GitLab instance that we within Nordic countries can use instead of GitHub or an international GitLab instance.